subsidy/forms.py

```diff
@@ -0,0 +1,14 @@
+import datetime
+from django import forms
+
+from .models import Subsidy
+
+
+class CreateNewSubsidy(forms.ModelForm):
+    class Meta:
+        model = Subsidy
+        fields = ['date', 'amount', 'name']
+        widgets = {
+            'date': forms.DateInput(attrs={'class': 'form-control', 'placeholder': datetime.date.today}, format='%Y-%m-%d'),
```

Comment:
I think it would be better to add an "initial" in date instead of a placeholder. It may be easier for people with low techology knowledge to understand, and in case you want to register a new subsidy with today's date (I guess it will be the mayority of cases) you don't have to modify that field.

Reply:
I will change it.